lib/tokenizers/model/bpe.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Tokenizers.Model.BPE do
+  @typedoc """
+  Options for model initialisation. All values are optional.
```

Comment:
I would change the docs here to say that "all options can be omitted". Because saying that they are optional can imply that they can be `nil`. WDYT?

Also, maybe we can validate the accepted options using [`Keyword.validate!/2`](https://hexdocs.pm/elixir/Keyword.html#validate!/2). WDYT?

Reply:
1. Y, mb it will be better. I'll change the entire codebase.
2. Values are validated by the mechanism of `NifTaggedEnum`. As keyword is basically a list of tuples2, each element can be parsed as `NifTaggedEnum`, while the keyword itself is a `Vec<T: NifTaggedEnum>`. Nif will rise with `:bad_arg`, mb it's not `:ok|:err` tuple, but `Keyword.validate!` will rise in the same way.